org.eclipse.jdt.ls.tests/projects/eclipse/java22/src/main/java/foo/bar/Foo.java

```diff
@@ -0,0 +1,9 @@
+package foo.bar;
```

Comment:
Instead of testing sealed classes could we just add a constructor with statements before `super()` ?

Reply:
Fixed.